lib/franklin/articles/article.ex

```diff
@@ -29,6 +29,16 @@ defmodule Franklin.Articles.Article do
     timestamps()
   end
 
+  def insert_changeset(article, attrs \\ %{}) do
+    Ecto.Changeset.cast(article, attrs, [
+      :id,
+      :body,
+      :published_at,
+      :slug,
+      :title
+    ])
+  end
```

Comment:
I had to add this since `insert` is the only verb I want to allow the user to submit an `id` value.